smart-send-logistics/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "smart-send",
+	"version": "1.0.0",
+	"author": "The WordPress Contributors",
```

Comment:
Please fix. See the main plugin file as reference.

Reply:
FIxed